src/main/java/io/toast/tk/automation/driver/web/SeleniumHelper.java

```diff
@@ -0,0 +1,94 @@
+package io.toast.tk.automation.driver.web;
+
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import io.toast.tk.automation.api.IMiniResult;
+import io.toast.tk.core.runtime.IWebElementDescriptor;
+
+public class SeleniumHelper {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS1118)